pkg/testing/integration_register.go

```diff
@@ -40,20 +40,21 @@ var _ = Suite(&PITRPostgreSQL{
 	},
 })
 
-// postgres app
-type PostgreSQL struct {
-	IntegrationSuite
-}
-
-var _ = Suite(&PostgreSQL{
-	IntegrationSuite{
-		name:      "postgres",
-		namespace: "postgres-test",
-		app:       app.NewPostgresDB("postgres", ""),
-		bp:        app.NewBlueprint("postgres", "", true),
-		profile:   newSecretProfile(),
-	},
-})
+// TODO: Add back after https://github.com/kanisterio/kanister/pull/1705 merges
```

Comment:
There's an easier way to disable tests :)
You could just remove them from this line - https://github.com/kanisterio/kanister/blob/master/build/integration-test.sh#L28

Reply:
Ah didn't see the shell script. Just followed the go stack up to the register